src/mumble/DBus.h

```diff
@@ -24,6 +24,8 @@ class MumbleDBus : public QDBusAbstractAdaptor {
 		void getCurrentUrl(const QDBusMessage &);
 		void getTalkingUsers(const QDBusMessage &);
 		void focus();
+		void setTransmitMode(unsigned int mode);
+		unsigned int getTransmitMode();
```

Comment:
I know that the other functions don't have documentation attached to them either, but I would ask you to add some nonetheless.

We use `///` doxygen comments, so if you could add a short documentation for both functions, that'd be great. It should look something like this:
```
/// Short description of what this function does
///
/// @param mode The new transmit mode (0 = continous, 1 = voice activity, 2 = PTT)
```